truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/impl/Accessor.java

```diff
@@ -277,6 +278,7 @@ protected Debugger createDebugger(Object vm, Instrumenter instrumenter) {
     @SuppressWarnings("unused")
     protected Closeable executionStart(Object vm, int currentDepth, Debugger debugger, Source s) {
         vm.getClass();
```

Comment:
### Thursday 04 of February 2016, 06:27:15 @ Christian Humer ###
> > @@ -277,6 +278,7 @@ protected Debugger createDebugger(Object vm,
> > Instrumenter instrumenter) {> 
> >      @SuppressWarnings("unused")
> >      protected Closeable executionStart(Object vm, int currentDepth,
> >      Debugger debugger, Source s) {>      
> >          vm.getClass();
> 
> Why vm.getClass()?

That is the cheapest check for non-null.
-jt